packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.controller.js

```diff
@@ -46,18 +48,66 @@ export default class PciProjectQuotaIncreaseController {
     );
     this.QUOTA_INCREASE_MODES = QUOTA_INCREASE_MODES;
     this.projectDescription = this.PciProject.getProjectInfo().description;
+    this.trackQuotaIncreasePopupDisplay();
   }
 
-  increaseQuota() {
+  isQuotaIncreaseConfirmButtonDisabled() {
+    if (
+      this.mode === QUOTA_INCREASE_MODES.BUY_CREDITS &&
+      this.serviceOptions.length > 0
+    ) {
+      return !(this.serviceOption && this.serviceOption.planCode);
+    }
+
+    if (this.mode === QUOTA_INCREASE_MODES.CONTACT_SUPPORT) {
+      return !this.issueTypeDescription;
+    }
+    return true;
+  }
+
+  trackQuotaIncreasePopupDisplay() {
     if (this.mode === QUOTA_INCREASE_MODES.BUY_CREDITS) {
-      return this.increaseQuotaByCredits();
+      return this.trackPage(`${TRACK.BASE}::${TRACK.SELECT_PLAN}`);
+    }
+    if (this.mode === QUOTA_INCREASE_MODES.CONTACT_SUPPORT) {
+      return this.trackPage(`${TRACK.BASE}::${TRACK.CONTACT_SUPPORT}`);
     }
+    return null;
+  }
 
-    return this.increaseQuotaBySupport();
+  trackPopupCancel() {
+    if (
+      this.mode === QUOTA_INCREASE_MODES.BUY_CREDITS &&
+      this.serviceOptions.length > 0
+    ) {
+      return this.trackClick(
+        `${TRACK.BASE}::${TRACK.SELECT_PLAN}::${TRACK.CANCEL}`,
+      );
+    }
+    if (this.mode === QUOTA_INCREASE_MODES.CONTACT_SUPPORT) {
```

Comment:
```suggestion
    else if (this.mode === QUOTA_INCREASE_MODES.CONTACT_SUPPORT) {
```

Reply:
Inside the "if" statement is a "return" statement. So there is no need to add an "else" statement.